tests/test_functional_post_install.py

```diff
@@ -127,6 +144,17 @@ def test_rose_fileinstall_uses_suite_defines(tmp_path):
             {
                 'test/rose-suite.conf': 'opts=a',
                 'test/opt/rose-suite-cylc-install.conf': '',
+                'test/opt/rose-suite-a.conf': '',
+                'test/opt/rose-suite-b.conf': '',
+                'test/opt/rose-suite-c.conf': '',
+                'ref/opt/rose-suite-cylc-install.conf': '!opts=b c\n',
+                'ref/rose-suite.conf': '!opts=a b c (cylc-install)',
```

Comment:
These two already in the dict from the previous commit?

Reply:
Ay